lalrpop/src/normalize/lower/mod.rs

```diff
@@ -427,7 +430,35 @@ impl<'s> LowerState<'s> {
                     let name_strs: Vec<_> = names.iter().map(AsRef::as_ref).collect();
                     name_strs.join(", ")
                 };
-                let action = action.replace("<>", &name_str);
+
+                let action = if action.matches("<>").count() > 1 {
+                    if action.matches("<>").count() != names.len() {
+                        // Here the error span will be based on the indices
```

Comment:
I know you're just keeping the existing variable name here, but isn't "indices" a wildly confusing variable name for this?  This is a vector of tuples, where the first tuple element is an index, and the second is an Anonymous Symbol.  Calling it "indices" is super weird.

I don't want to hold up this PR over that, since it's really an issue in the existing code (although that existing issue makes this comment pretty confusing IMO).  But I'm just noting it for reference and I might make a follow-up PR to change this name unless you see a reason to justify the existing name choice?